TrafficCapture/dockerSolution/src/main/docker/elasticsearchTestConsole/simpleDocumentQuery.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python
+import requests
+from datetime import datetime
+import argparse
+import os
+import urllib3
+import time
+from collections import deque
+from concurrent import futures
+import logging
+
+# Suppress only the single InsecureRequestWarning from urllib3 needed for this script
+urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
+
+# Configure logging
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+# Set the base URLs from the environment variables or use default values
+source_url_base = os.getenv('SOURCE_DOMAIN_ENDPOINT', 'https://elasticsearch:9200')
+target_url_base = os.getenv('MIGRATION_DOMAIN_ENDPOINT', 'https://opensearchtarget:9200')
+source_username = 'admin'
+source_password = 'admin'
+target_username = 'admin'
+target_password = 'myStrongPassword123!'
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--source-endpoint",
+        help="Source cluster endpoint e.g. http://source.elb.us-west-2.amazonaws.com:9200."
+    )
+    parser.add_argument(
+        "--target-endpoint",
+        help="Target cluster endpoint e.g. http://target.elb.us-west-2.amazonaws.com:9200."
+    )
+    parser.add_argument(
+        "--source-username",
+        help="Source cluster username.",
+        default=source_username
+    )
+    parser.add_argument(
+        "--source-password",
+        help="Source cluster password.",
+        default=source_password
+    )
+    parser.add_argument(
+        "--target-username",
+        help="Target cluster username.",
+        default=target_username
+    )
+    parser.add_argument(
+        "--target-password",
+        help="Target cluster password.",
+        default=target_password
+    )
+    parser.add_argument(
+        "--source-no-auth",
+        action='store_true',
+        help="Flag to provide no auth in source requests."
+    )
+    parser.add_argument(
+        "--target-no-auth",
+        action='store_true',
+        help="Flag to provide no auth in target requests."
+    )
+    parser.add_argument(
+        "--no-clear-output",
+        action='store_true',
+        help="Flag to not clear the output before each run. Helpful for piping to a file or other utility."
+    )
+    return parser.parse_args()
```

Comment:
Any reason why all these options (like customizable user/pass) are available here and not for the generator?

Reply:
I'll go ahead and add those options